docs/operating-scylla/nodetool-commands/scrub.rst

```diff
@@ -87,4 +109,50 @@ Scrub **a** specific table (mytable) in a keyspace (mykeyspace) in VALIDATE mode
 
    > nodetool scrub -m VALIDATE --no-snapshot mykeyspace mytable
 
+Scrub **a** specific table (mytable) in a keyspace (mykeyspace) in SEGREGATE mode dropping unfixable SSTables
+
+.. code-block:: shell
+
+   > nodetool scrub -m SEGREGATE --drop-unfixable-sstables mykeyspace mytable
+
+Procedures for Removing Bad SSTables
+.....................................
+
+Method 1: Quarantine and Drop
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Step 1**: Run scrub in VALIDATE mode to identify and quarantine corrupted SSTables:
+
+.. code-block:: shell
+
+   > nodetool scrub -m VALIDATE keyspace_name table_name
+
+This will move corrupted SSTables to a ``quarantine`` subdirectory.
+
+**Step 2**: Drop the quarantined SSTables using :doc:`dropquarantinedsstables </operating-scylla/nodetool-commands/dropquarantinedsstables>`:
```

Comment:
Should have another step before this: move all or a sample of sstable aside, for R&D to have a look at them. Or at consider doing so.